maintainers/scripts/haskell/merge-and-open-pr.sh

```diff
@@ -53,6 +53,10 @@ if ! gh auth status 2>/dev/null ; then
   die "You must setup the \`gh\` command.  Run \`gh auth login\`."
 fi
 
+# Make sure this is configured before we start doing anything
+push_remote="$(git config branch.haskell-updates.pushRemote \
+  || die "branch.haskell-updates.pushRemote is not configured")"
```

Comment:
Could you add the literal git cli command you use to set `branch.haskell-updates.pushRemote` here in this error message?